src/llm_models/llm_model.rs

```diff
@@ -69,4 +70,16 @@ pub trait LLMModel {
         //To be safe we go with smaller of the numbers
         std::cmp::min(max_requests_from_rpm, max_requests_from_tpm)
     }
+    ///Returns the default temperature to be used by the model
+    fn get_default_temperature(&self) -> f32 {
+        0f32
+    }
+    ///Returns the normalized temperature for the model
+    //Input should be a 0-100 number representing the percentage of max temp for the model
+    fn get_normalized_temperature(&self, relative_temp: u32) -> f32 {
+        //Assuming 0-1 range for most models. Different ranges require model-specific implementations.
+        let min = 0u32;
```

Comment:
most models have a 0-1 range for temperature so this default implementation addresses most of them.